.github/workflows/deploy.yml

```diff
@@ -33,188 +25,39 @@ jobs:
+      - name: Setup Node and Dependencies
+        uses: ./.github/actions/setup-node
 
       - name: Run security audit
         run: yarn npm audit
 
   snyk:
     name: Snyk Security Scan
-    uses: ministryofjustice/laa-reusable-github-actions/.github/workflows/snyk.yml@73a98b3336f71a21c46e6b588cd3afaaab052649
-    with:
-      tag: "your-repo-name"
-    secrets:
-      snyk_token: ${{ secrets.SNYK_TOKEN }}
-
-  build-image:
-    name: Build Docker Image
     runs-on: ubuntu-latest
-    permissions:
-      id-token: write
-      contents: read
     steps:
       - name: Checkout code
-        uses: actions/checkout@v4
+        uses: actions/checkout@v5
 
-      - name: Set up AWS credentials for ECR and Kubernetes
-        uses: aws-actions/configure-aws-credentials@v4.2.1
-        with:
-          role-to-assume: ${{ secrets.ECR_ROLE_TO_ASSUME }}
-          aws-region: ${{ vars.ECR_REGION }}
+      - name: Build local Docker image
+        working-directory: ${{ github.workspace }}
+        run: |
+          docker build -t laa-manage-your-civil-cases ./
+          docker save laa-manage-your-civil-cases -o image.tar
 
-      - uses: aws-actions/amazon-ecr-login@062b18b96a7aff071d4dc91bc00c4c1a7945b076
-        id: login-ecr
-
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@e468171a9de216ec08956ac3ada2f0791b6bd435
-        with:
-          install: true
-
-      - name: Cache Docker layers
-        uses: actions/cache@v4
+      - name: Run Snyk to scan Docker image from tarball
+        uses: snyk/actions/docker@master
         with:
-          path: /tmp/.buildx-cache
-          key: ${{ runner.os }}-buildx-${{ github.sha }}
-          restore-keys: |
-            ${{ runner.os }}-buildx-
-
-      - name: Build and push image with local action
-        uses: ./.github/actions/build_and_push
-        with:
-          image_registry: ${{ secrets.ECR_REGISTRY_URL }}
-          image_repo: ${{ vars.ECR_REPOSITORY }}
-          dockerfile_path: Dockerfile
-          image_tag: ${{ github.sha }}
+          image: image.tar
+          args: --docker --severity-threshold=high
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
 
   mocha-unit-tests:
     name: Mocha Unit Tests
     uses: ./.github/workflows/mocha.yml
-    needs:
-      - build-image
-    # Only run tests when deploying to UAT
-    if: success() && github.event_name != 'workflow_dispatch' || (github.event_name == 'workflow_dispatch' && github.event.inputs.environment == 'uat' || github.event.inputs.environment == null)
 
   playwright:
     name: Playwright Tests
     uses: ./.github/workflows/playwright.yml
-    needs:
-      - build-image
-    # Only run tests when deploying to UAT
-    if: success() && github.event_name != 'workflow_dispatch' || (github.event_name == 'workflow_dispatch' && github.event.inputs.environment == 'uat' || github.event.inputs.environment == null)
-  
-  deploy-uat:
-    environment: uat
```

Comment:
Question - Do we need to document how to deploy this now that we are removing the template actions to do so? 

Also the name of the file will be confusing as it's not a `deploy` action but a `ci-pipeline` action.